Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45768&attempt_id=5fe455ac-2d02-449c-9f65-6e3d34e6b02c)

Attempted: perlPackages.TextUnaccent

<details><summary>Partial log (click to expand)</summary><p>

```
-------------------
t/unac.t (Wstat: 11 Tests: 0 Failed: 0)
  Non-zero wait status: 11
  Parse errors: Bad plan.  You planned 4 tests but ran 0.
Files=1, Tests=0,  0 wallclock secs ( 0.02 usr  0.00 sys +  0.01 cusr  0.00 csys =  0.03 CPU)
Result: FAIL
Failed 1/1 test programs. 0/0 subtests failed.
make: *** [Makefile:1025: test_dynamic] Error 255
builder for '/nix/store/jnpynv08v5jshnn87kxdg273ybd2mb8k-perl5.28.0-Text-Unaccent-1.08.drv' failed with exit code 2
error: build of '/nix/store/jnpynv08v5jshnn87kxdg273ybd2mb8k-perl5.28.0-Text-Unaccent-1.08.drv' failed
```
</p></details>

--- a/pkgs/top-level/perl-packages.nix
+++ b/pkgs/top-level/perl-packages.nix
@@ -16575,12 +16575,11 @@ let
       url = mirror://cpan/authors/id/L/LD/LDACHARY/Text-Unaccent-1.08.tar.gz;
       sha256 = "0avk50kia78kxryh2whmaj5l18q2wvmkdyqyjsf6kwr4kgy6x3i7";
     };
-    buildInputs = stdenv.lib.optionals stdenv.isDarwin [ pkgs.libiconv ];
     # https://rt.cpan.org/Public/Bug/Display.html?id=124815
     NIX_CFLAGS_COMPILE = [ "-DHAS_VPRINTF" ];
     meta = {
       maintainers = with maintainers; [ ];
-      platforms   = stdenv.lib.platforms.unix;
+      platforms = stdenv.lib.platforms.linux;
     };
   };
 

